Packs/McAfee-MAR/Integrations/McAfee-MAR/McAfee-MAR.py

```diff
@@ -313,6 +321,7 @@ def main():
             demisto.info('######## executing test command ########')
             test()
             demisto.results('ok')
+            delete_files()
```

Comment:
Use try and finally in the main function.
No need to call delete_files every command call